### ISSUE TITLE
gattlib version requirement is too strict in setup.py

### ISSUE THREAD

**Issue by Joshuaalbert** [mbientlab/MetaWear-SDK-Python#5]
When I attempt to install via `python setup.py install` setup proceeds to clone and install `gattlib-0.20171119` however in metawear's setup.py it requires gattlib==0.20171002 so it will fail. Changing this to gattlib>=0.20171002 fixes this problem.

**Comment by scaryghost** [comment]
I've reverted the version string in our gattlib fork to fix the Python SDK build.

The problem with using range based versioning with gattlib is that it does not follow semantic versioning.  Furthermore, gattlib is not under active development so its safe to assume that the version string will not be changing for a long time (barring this recent mix-up).

**Comment by onshoremanover** [comment]
I have successfully installed gattlib-0.20171002, but when I pip install metawear / or also pymetawear /or metawear --process-dependency-links, I get the same error:  Could not find a version that satisfies the requirement gattlib==0.20171002 (from metawear) (from versions: 0.20150119.linux-x86_64, 0.20150119, 0.20150130, 0.20150131, 0.20150805)
No matching distribution found for gattlib==0.20171002 (from metawear).
What did I wrong? I installed all the dependencies, maybe I additionally have an older or newer version of gattlib, could that be the problem?
